gradoop-examples/src/main/resources/data/gdl/tc.gdl

```diff
@@ -0,0 +1,102 @@
+tc[
+// vertex space
+(alice:Person {name:"Alice", country: "Germany", state: "Saxony", city:"Leipzig"})
+(bob:Person {name:"Bob", country: "Germany", state: "Saxony", city:"Leipzig"})
+(carol:Person {name:"Carol", country: "Germany", state: "Saxony", city:"Riesa"})
+(dave:Person {name:"Dave", country: "Germany", state: "Saxony", city:"Dresden"})
+(eve:Person {name:"Eve", country: "Germany", state: "Bavaria", city:"Ingolstadt"})
+(frank:Person {name:"Frank", country: "Germany", state: "Bavaria", city:"Nuremberg"})
+(bjarne:Person {name:"Bjarne", country: "Germany", state: "Brandenburg", city: "Cottbus"})
+(donald:Person {name:"Donald", country: "Germany", state: "Bavaria", city:"Regensburg"})
+(marcus:Person {name:"Marcus", country: "Germany", state: "Bavaria", city:"Ingolstadt"})
+(ronald:Person {name:"Ronald", country: "Germany", state: "Saarland", city:"Neunkirchen"})
+(alexander:Person {name:"Alexander", country: "Germany", state: "Bavaria", city:"Nuremberg"})
+(david:Person {name:"David", country: "Germany", state: "Brandenburg", city:"Leipzig"})
+(doris:Person {name:"Doris", country: "Germany", state: "Brandenburg", city:"Berlin"})
+(martina:Person {name:"Martina", country: "Germany", state: "Bavaria", city:"Munich"})
+(christina:Person {name:"Christina", country: "Germany", state: "Saarland", city:"Saarbruecken"})
+(petra:Person {name:"Petra", country: "Germany", state: "Thuringia", city:"Gera"})
+(kevin:Person {name:"Kevin", country: "Germany", state: "Saarland", city:"Saarbruecken"})
+(ole:Person {name:"Ole", country: "Germany", state: "Thuringia", city:"Jena"})
+(vincent:Person {name:"Vincent", country: "Germany", state: "Thuringia", city:"Jena"})
+(maria:Person {name:"Maria", country: "Germany", state: "Saxony", city:"Dresden"})
+(marta:Person {name:"Marta", country: "Germany", state: "Saxony", city:"Dresden"})
+(lisa:Person {name:"Lisa", country: "Germany", state: "Thuringia", city:"Erfurt"})
+(linda:Person {name:"Linda", country: "Germany", state: "Thuringia", city:"Erfurt"})
```

Comment:
Maybe out-migrate Lisa, Linda and Rica to the USA to have at least a few person from a different country.